src/Repository/SessionRepository.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Session;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * @method Session|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Session|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Session[]    findAll()
+ * @method Session[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class SessionRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Session::class);
+    }
+
+    /**
+     * @param $day
+     * @return mixed
+     */
+    public function findByDayField($day)
```

Comment:
kuo visiems akademijos studentams taip baisiai nepatinka standartinis repositorijos metodas `$repo->findBy(['day' => 'some day']);`??
Nieko kodint nereikia ir its the way it should be done